bot.py

```diff
@@ -109,6 +113,64 @@ def get_members_by_role(role):
                 members.append(member)
                 break
     return(members)
+    
+def get_quotes():
+    try:
+        with open(os.path.join(sys.path[0], 'quotes.dat'), 'rb') as fp:
+            return pickle.load(fp)
+    except FileNotFoundError:
+        quotes = []
+        return quotes
+        
+def create_quote_image(quote, name):
+    text = "\"{}\"".format(quote)            
+    name = "- {}".format(name)
+    file = "images/{}.jpg".format(random.randint(1,50))
```

Comment:
Instead of blindly assuming images 1-5.jpg exist, how about we get a list of images in the folder and then randomly select one? That way we can add/remove images as needed easily.